Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -435,69 +448,85 @@ public void indexConcepts(String userName)
             int numberOfRecordsPerPage) throws LuceneException, IllegalAccessException {
 
         Map<String,Analyzer> analyzerPerField = new HashMap<>();
-        
-        if (operator == null) {
-            operator = SearchParamters.OP_AND;
+        BooleanClause.Occur occur = BooleanClause.Occur.SHOULD;
+        if (operator == null || operator.equalsIgnoreCase(SearchParamters.OP_AND)) {
+            occur = BooleanClause.Occur.MUST;
         }
 
-        StringBuffer queryString = new StringBuffer();
-        int firstEntry = 1;
-
         java.lang.reflect.Field[] fields = ConceptEntry.class.getDeclaredFields();
+
+        for (java.lang.reflect.Field field : fields) {
+            LuceneField luceneFieldAnnotation = field.getAnnotation(LuceneField.class);
+            if (luceneFieldAnnotation != null) {
+                if (!luceneFieldAnnotation.isShortPhraseSearchable() && !luceneFieldAnnotation.isTokenized()) {
+                    // If the field is not tokenzied, then the field needs to be
+                    // analyzed using a whitespaceanalyzer, rather than standard
+                    // analyzer. This is because for non tokenized strings we
+                    // need exact matches and not all the nearest matches.
+                    analyzerPerField.put(luceneFieldAnnotation.lucenefieldName(), whiteSpaceAnalyzer);
+                } if(luceneFieldAnnotation.isShortPhraseSearchable()) {
+                    analyzerPerField.put(luceneFieldAnnotation.lucenefieldName() + LuceneFieldNames.UNTOKENIZED_SUFFIX,
+                            keywordAnalyzer);
+                }
+            }
+
+        }
+
+        PerFieldAnalyzerWrapper perFieldAnalyzerWrapper = new PerFieldAnalyzerWrapper(standardAnalyzer,
+                analyzerPerField);
+
+        QueryBuilder qBuild = new QueryBuilder(perFieldAnalyzerWrapper);
+        BooleanQuery.Builder builder = new BooleanQuery.Builder();
+
         for (java.lang.reflect.Field field : fields) {
             SearchField search = field.getAnnotation(SearchField.class);
             LuceneField luceneFieldAnnotation = field.getAnnotation(LuceneField.class);
             if (search != null) {
                 String searchString = fieldMap.get(search.fieldName());
+                if (searchString != null) {
+                    if (containsWildCardCharacters(searchString)) {
```

Comment:
just add another property to the annotation to enable a field for wildcard search and then use that here.